dev/benchmarks/microbenchmarks/lib/common.dart

```diff
@@ -33,6 +77,28 @@ class BenchmarkResultPrinter {
     _results.add(_BenchmarkResult(description, value, unit, name));
   }
 
+  /// Adds a benchmark result to the list of results and a probability of that
+  /// result.
+  ///
+  /// The probably is calculated as the probability that the mean is +- 5% of
+  /// the true value.
+  ///
+  /// See also [addResult].
+  void addResultStatistics({
+    required String description,
+    required List<double> values,
+    required String unit,
+    required String name,
+  }) {
+    final double mean = _doMean(values);
+    final double stddev = _doStddev(values, mean);
+    const double margin = 0.05;
+    final double probability = _doProbability(mean: mean, stddev: stddev, margin: margin);
+    _results.add(_BenchmarkResult(description, mean, unit, name));
+    _results.add(_BenchmarkResult('$description - probability margin of error $margin', probability,
+        'percent', '${name}_probability'));
```

Comment:
Consider adding the median here

Reply:
And maybe the min/max?

Reply:
Those values are going to be really noisy so I don't know if we want them showing up in skia perf graphs.  The probability of %5 error is something that should be stable and if it increases it's a regression.  It means our benchmarks have become less reliable for some reason.